package.json

```diff
@@ -155,7 +155,7 @@
     "react-native": "0.72.3",
     "react-native-builder-bob": "^0.18.3",
     "react-native-gesture-handler": "^2.12.0",
-    "react-native-macos": "^0.71.0-0",
+    "react-native-macos": "^0.72.0",
```

Comment:
Is this dependency actually needed in the root of our repo?

Reply:
It turned out it is not. I'm about to push appropriate changes.